src/main/resources/application.yaml

```diff
@@ -62,9 +61,50 @@ hapi:
     ### Flag is false by default, can be passed as command line argument to override.
     cr:
       enabled: false
+      caregaps:
+        reporter: "default"
+        section_author: "default"
+      cql:
```

Comment:
Couldn't all this be left as commented out and use default values?

Reply:
Yes, definitely.  I believe this was done more to illustrate that these options are available.